components/infobox/wikis/apexlegends/infobox_league_custom.lua

```diff
@@ -230,7 +214,7 @@ function CustomLeague:getWikiCategories(args)
 	if String.isNotEmpty(args.participants_number) then
 		table.insert(categories, 'Individual Tournaments')
 	end
-	if String.isNotEmpty(args.eatier) or args['ea-sponsored'] == 'true' then
+	if args['ea-sponsored'] == 'true' then
```

Comment:
Seems like this could be also mapped to publishertier input that gets filtered for storage

Reply:
sadly they are used in parallel in a way that would fuck with publishertier storage
e.g. for `|ea-sponsored=true|algstier=Online` it currently would store empty string while still highlighting stuff and adding an icon on the infobox itself
don't like that behaviour at all but imo OOS for this PR to fix the entirety of apex infobox league weirdness